index.js

```diff
@@ -3,6 +3,29 @@ var path = require('path');
 var eachAsync = require('each-async');
 var globby = require('globby');
 var cpFile = require('cp-file');
+var pathIsAbsolute = require('path-is-absolute');
+
+function preprocessSrcPath(srcPath, opts) {
+	if (!pathIsAbsolute(srcPath) && opts.cwd) {
+		srcPath = path.join(opts.cwd, srcPath);
+	}
+	return srcPath;
+}
+
+function preprocessDestPath(srcPath, dest, opts) {
+	var basename = path.basename(srcPath);
+	var dirname = path.dirname(srcPath);
+
+	if (!pathIsAbsolute(dest) && opts.cwd) {
+		dest = path.join(opts.cwd, dest);
+	}
```

Comment:
I think you can use `path.resolve` instead so don't have to check if it's absolute.